docker/cheqd_node/mainnet_launch/.env

```diff
@@ -0,0 +1,10 @@
+
+P2P_SEEDS=258a9bfb822637bfca87daaab6181c10e7fd0910@seed1.eu.cheqd.net:26656,f565ff792b20977face9817df6acb268d41d4092@seed2.eu.cheqd.net:26656,388947cc7d901c5c06fedc4c26751634564d68e6@seed3.eu.cheqd.net:26656,9b30307a2a2819790d68c04bb62f5cf4028f447e@seed1.ap.cheqd.net:26656,debcb3fa7d40e681d98bcc7d22278fd58a34b73a@144.76.183.180:1234,abd4be300be882ae9a69ab0959260afe8871f7a6@165.232.167.156:26656,fd17fe46e8b69bfa006b3fba53cfc9df2b8f9512@161.35.177.151:26656
+
+RPC_LADDR=tcp://0.0.0.0:26657
+
+# The next parameters should be changed by user
+NODE_MONIKER=<name-of-node>
+
+OUT_MOUNT_POINT=/path/to/data/on/the/host
```

Comment:
Consider mount points (used above) vs using volumes in general. Volumes will ensure that there's consistent permissions and ownership that can be applied to the contents. See this for a comparison: https://docs.docker.com/engine/reference/commandline/service_create/#add-bind-mounts-volumes-or-memory-filesystems